README.md

```diff
@@ -80,6 +80,20 @@ spec:
   credentialsSecret: internal-mirror
 ```
 
+If you want to adjust the amount of storage allocated, you can add a `persistentVolumeClaimSpec` key to the spec. The value should be the same as a [persistentVolumeClaim](https://v1-8.docs.kubernetes.io/docs/api-reference/v1.8/#persistentvolumeclaimspec-v1-core) object. e.g:
+```yaml
+apiVersion: k8s.osp.tech/v1
+kind: RegistryMirror
+metadata:
+  name: internal
+spec:
+  upstreamUrl: hub.docker.io
+  persistentVolumeClaimSpec:
```

Comment:
not a fan of this field name, why the trailing `Spec`?

Reply:
Hmm, I have no idea tbh.

How would `volumeClaimTemplate` be? (That's in line with what it is in StatefulSet)

Reply:
Ok, let's do it like Prometheus then - which is `volumeClaimTemplate`, and the same spec I currently have?

Reply:
How's that?